indico/modules/events/contributions/controllers.py

```diff
@@ -95,8 +79,40 @@ class RHContributions(RHManageContributionsBase):
     """Display contributions management page"""
 
     def _process(self):
+        contrib_report_config = reporter.get_report_config(self.event_new.id)
+        if 'config' in request.args:
```

Comment:
OK, I get your point. But I use `get_report_config` in many places where I don't need the second element of the tuple. Should I assign it to `_`? Then I would override the i18n `_`...